README.md

```diff
@@ -31,262 +31,217 @@ Be sure that the installer path reflects your WordPress plugin directory.
-	 */
-	protected function setItems($items) {
-		$this->items = $items;
-	}
-
-	/**
-	 * Start the batch job by compiling the batch
-	 * @return string JSON string
-	 */
-	public function run() {
-		
-		//Update the last run date to be displayed in the UI
-		$this->updateLastRunDate();
-
-		//Get a large batch of something, posts, users, DB records, etc. For this example we get all the 'post' post type ID's in a numerically indexed array.
-		global $wpdb;
-		$postIds = $wpdb->get_results("SELECT ID from {$wpdb->prefix}posts WHERE post_type = 'post'", ARRAY_N);
-
-		//Set the items for later reference
-		$this->setItems($postIds);
-
-		//Output JSON for AJAX
-		wp_send_json_success([
-			'items' => $this->getItems()
-		]);
-	}
-
-	/**
-	 * Process an individual batch job item
-	 * @return array Response data
-	 */
-	public function runItem() {
-		if( !wp_doing_ajax() ) {
-			return;//You should probably log and throw an error.
-		}
-
-		//Grab our item from the AJAX request
-		$postId = $_REQUEST['item'];
-
-		//Setup an empty array to hold our response. As batch jobs should only be run in the WP admin by authenticated users, you can use this response to output information in the browser console.
-		$response = [];
-
-		//We're only expecting post ID's, so let's do a quick sanity check. If this fails in real life, you should probably log it out for further analysis.
-		if( !$postId || !is_numeric($postId) ) {
-			$response['reason'] = __FUNCTION__ . ' requires a numeric post ID.';
-			wp_send_json_error($response);
-		}
-
-		$response['postId'] = $postId;
-
-		//Let's add some post meta for each post
-		$meta_key = 'batch_meta_key';
-		$meta_value = 'Your batch meta value';
-		$postMetaId = add_post_meta($postId, $meta_key, $meta_value);
-
-		if( !$postMetaId ) {
-			$response['reason'] = 'Could not save post meta.';
-			wp_send_json_error($response);
-		}
-
-		$response['postMetaId'] = $$postMetaId;
-		$response['reason'] = "Post meta for $postId was updated to '$meta_value'.";
-
-		//Send the JSON success message
-		wp_send_json_success($response);
-	}
-}
\ No newline at end of file
+/**
+ * Add or remove table columns from the Batch Jobs admin page.
+ *
+ * @param array $columns Registered table columns.
+ * @return array
+ */
+add_filter('ramsey-batch-table-columns', function ($columns) {
+    $columns[] = [
+        'id' => 'columnHeaderID',
+        'name' => 'Custom Column Name',
+        'contentKey' => 'jobArrayKeyFromWhereToPullContent',
+    ];
+    return $columns;
+});
+```
+
+The content of that newly registered column can be accessed by adding a new key to the job itself, using the `ramsey-batch-jobs` filter. This key must match the `contentKey` registered above. 
+
+```php
+add_filter('ramsey-batch-jobs', function ($jobs, $instance) {
+    $jobs['MyAppNamespace\Batch\YourJob']['jobArrayKeyFromWhereToPullContent'] = '
```

Comment:
🗣 Make it match above suggestion.

`$jobs['MyAppNamespace\Batch\YourJob']['myColumnNameOrSlug'] = '`